src/main/java/org/candlepin/subscriptions/tally/MetricUsageCollector.java

```diff
@@ -360,10 +360,7 @@ private void addBucketsFromEvent(Host host, Event event, Optional<TagMetaData> s
         Optional.ofNullable(event.getBillingProvider())
             .map(Event.BillingProvider::toString)
             .map(BillingProvider::fromString)
-            .orElse(
-                serviceTypeMeta
-                    .map(TagMetaData::getDefaultProvider)
```

Comment:
Why was this was removed?

Reply:
@kflahert, we don't need per-product defaults for the billing provider. It was extra unneeded complexity that was adding noise when we were debugging/tracing how things work.